### ISSUE TITLE
Training may begin encoding text embeds when they disappear or were missing

### ISSUE THREAD

**Issue by bghira** [bghira/SimpleTuner#277]
After #275, a new issue is introduced where a training run will potentially begin encoding text embeds during training.

Previously, an error would emit when loading a text embed fails during training. Instead, we now catch the error and ensure we encode a new embed.

Existing flags may be useful, otherwise a new flag or StateTracker element are needed to force an error to emit here instead of encoding a prompt during training.
